addons/mail/static/src/models/rtc_call_participant_card/rtc_call_participant_card.js

```diff
@@ -111,7 +111,7 @@ function factory(dependencies) {
          * @returns {boolean}
          */
         _computeIsTalking() {
-            return Boolean(this.rtcSession && this.rtcSession.isTalking && !this.rtcSession.isMuted);
+            return Boolean(this.rtcSession && this.rtcSession.isTalking && !this.rtcSession.isMute);
```

Comment:
It was already there before PR, but this shows the concept of "is talking" is not well defined. The 2 models `RtcSession` and `RtcCallParticipantCard` are heavily related, so it's strange that `isTalking` differs.

As I said in previous comment, it doesn't make sense having `isTalking` while `isMute`.

Reply:
Regarding the two commentso n isTalking, we also talked a bit about it with @seb-odoo we could bind `track.enabled` to `isTalking`, I'll make a PR for master with these changes (and some other changes that were mentioned in this PR), I just created task-2726154 for that.